index.html

```diff
@@ -0,0 +1,70 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+</head>
+
+<body> 
+    <header class="navigation">
+        <nav class="toolbar-container">
+            <div class="logo"><a href="#welcome">Welcome</a></div>
+            <div>
+                <a class="icon" ><a href="#menu-bar">
+                <i class="fa fa-bars"></i></a>
+            </div>
+        </nav>
+    </header>
+    <section class="headline back-image">
+        <div class="primary-headline" id="desktop-primary">
+            <h1>Hey there. I'm <br> Hiwot<span class="h1color"><br>
+             I'm a software developer</span>
+            </h1>
+        </div>
+         <div>
+            <p class="supporting-text">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div>
+            <ul class="socialt">
+                <li> <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/vector.png" /></a> </li>
+                <li> <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/angellist.svg" /></a> &nbsp; </li>
+            </ul> 
+        </div>
+        <div class="scroll">
+            <img src="image/Disabled.png" alt="Scroll down" >
+        </div>
+    </section>
+
+    <section class="headline-desk-container">
+        <div id="desktop-headline">
+            <h1>Hey there. I'm Hiwot<span class="h1color"><br>
+             I'm a software developer</span>
+            </h1>
+            <p>  I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+            </p>
+        <div>
+            <ul id="desktop-socialt">
+                <li> <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/vector.png" /></a> </li>
+                <li> <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/angellist.svg" /></a> &nbsp; </li>
+                <li> <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp; </li>
+            </ul> 
+        </div>
+        <div id="scroll">
+            <img src="image/Disabled.png" alt="Scroll down" >
+        </div>
+    </section>
```

Comment:
- [ ] Kindly consider removing this second section. All the content here already exists in the first section and no need to hardcode it again. If it's for the desktop version the CSS can handle that .